virttest/utils_windows/drive.py

```diff
@@ -1,18 +1,28 @@
 """
 Windows drive utilities
 """
-
-from virttest import utils_misc
-
-from . import wmic
+import re
 
 
 def _logical_disks(session, cond=None, props=None):
-    cmd = wmic.make_query("LogicalDisk", cond, props, get_swch=wmic.FMT_TYPE_LIST)
-    out = utils_misc.wait_for(
-        lambda: wmic.parse_list(session.cmd(cmd, timeout=120)), 240
-    )
-    return out if out else []
+    c_name, c_value = cond.split("=")
+    cmd = (
+        'powershell -command "Get-CimInstance -ClassName Win32_LogicalDisk | Where-Object {$_.%s -like %s}'
```

Comment:
The disk needs to be under single quotes, same you've done in utils_netperf.py
```suggestion
        'powershell -command "Get-CimInstance -ClassName Win32_LogicalDisk | Where-Object {$_.%s -like '%s'}'
```

Reply:
Hi @mcasquer Thank you for your suggestion, actually, `c_value` already has quotes, you can check line37.